sda-commons-cacertificates/src/test/java/org/sdase/commons/cacertificates/CaCertificatesConfigurationTest.java

```diff
@@ -14,9 +14,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
-@SpringBootTest(
-    classes = TestApp.class,
-    properties = {"sda.caCertificates.certificatesDir=src/test/resources"})
+@SpringBootTest(classes = TestApp.class)
```

Comment:
Could we duplicate this test to verify bean creation with and without config?

Reply:
done 👍